test/MiddlewareListenerTest.php

```diff
@@ -106,6 +127,36 @@ public function testSuccessfullyDispatchesMiddleware()
         $this->assertEquals('Test!', $return->getBody());
     }
 
+    /**
+     * Stratigility v2 does not support PHP 8
+     * @requires PHP <8
+     */
+    public function testDispatchingMiddlewareTriggersDeprecation(): void
+    {
+        error_reporting($this->errorReporting & E_USER_DEPRECATED);
+        $this->expectDeprecation();
+        $this->expectDeprecationMessage('use laminas/laminas-mvc-middleware');
```

Comment:
For suggested wording changes test expectation needs to be updated as well
```suggestion
        $this->expectDeprecationMessage('use the laminas/laminas-mvc-middleware package');
```